src/System.Windows.Extensions/src/System/Drawing/IconConverter.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Globalization;
+using System.IO;
+
+namespace System.Drawing
+{
+    public class IconConverter : ExpandableObjectConverter
+    {
+        public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
+        {
+            return (sourceType == typeof(byte[]));
+        }
+
+        public override bool CanConvertTo(ITypeDescriptorContext context, Type destinationType)
+        {
+            return destinationType == typeof(byte[]) || destinationType == typeof(string);
```

Comment:
IconConverter in .NET Framework supports converting to Bitmap and Image -- can we add that support?  https://referencesource.microsoft.com/#System.Drawing/commonui/System/Drawing/IconConverter.cs,55